app/(white_navbar)/explore/RetroPGF4/_component/Tab/component/AllocationChart.tsx

```diff
@@ -0,0 +1,116 @@
+import Highcharts from 'highcharts'
+import HighchartsReact from 'highcharts-react-official'
+import { FC } from 'react'
+import { mapImpactMetrics } from '../../../[name]/Text'
+
+interface AllocationChartProps {
+  metricSelected: any[]
```

Comment:
More proper type! even if the metrics are vary based on each badgeholder but we can still create a type out of it.